administrator/components/com_scheduler/tmpl/tasks/empty_state.php

```diff
@@ -15,7 +15,7 @@
 $displayData = [
 	'textPrefix' => 'COM_SCHEDULER',
 	'formURL' => 'index.php?option=com_scheduler&task=task.add',
-	'helpURL' => 'https://github.com/joomla-projects/soc21_website-cronjob',
+	'helpURL' => 'https://docs.joomla.org/J4.x:Task_Scheduler',
```

Comment:
```suggestion
	'helpURL' => 'https://docs.joomla.org/Special:MyLanguage/J4.x:Task_Scheduler',
```
This addition means that the docs site will automatically change to the users language